core/src/thread_parker/windows/mod.rs

```diff
@@ -40,13 +41,20 @@ impl Backend {
         }
 
         // Try to create a new object
```

Comment:
So this isn't exactly your fault, but I just realized that this comment is nonsense. We've already created the `Backend`, now we just need to try to make it the global backend.